hazelcast/src/test/java/com/hazelcast/util/executor/CancellableDelegatingFutureTest.java

```diff
@@ -35,17 +34,15 @@ public void testInnerFutureThrowsCancellationExceptionWhenOuterFutureIsCancelled
             try {
                 future.getFuture().get();
                 fail();
-            } catch (ExecutionException expected) {
-                assertTrue(expected.getCause() instanceof CancellationException);
+            } catch (CancellationException expected) {
```

Comment:
You can e.g. match for the root cause if you have nested exceptions (just search for the usage of  `RootCauseMatcher`). Or you can match on exception messages (which is not nice, but if it's necessary you can do it).